include/objects/disk.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "object.h"
+
+namespace objects
+{
+
+struct Disk : Object
+{
+    float radius = 1.0f;
+    lmath::Normal normal;
```

Comment:
normal; -> normal{}; because it doesn't have constructor that takes 0 params, so all the variables in structure with {} will be zero by default